docs/Pipeline-Documentation.md

```diff
@@ -1,44 +1,52 @@
 
 ## Detection
 
 Best for a series of images that have no temporal relationship, such as arial photography of multiple scenes.  Also preferred if you only care about aggregate date for the dataset, such as max occurrences of an object per scene.
 
 | Pipeline | Use case |
 | -------- | -------- |
-| <pre>arctic seal eo yolo</pre> | Detector for color imagery |
-| <pre>arctic seal ir yolo</pre>  | Detector for infrared |
-| <pre>em tuna</pre>  | Detector for identifying individual features of tuna |
+| <pre>arctic seal eo yolo</pre> | detector for color imagery |
+| <pre>arctic seal ir yolo</pre>  | detector for infrared |
+| <pre>em tuna</pre>  | detector for identifying individual features of tuna |
 | <pre>fish without motion</pre>  | simple single-class fish detector |
 | <pre>generic proposals</pre>  | generic object detector |
-| <pre>motion</pre>  | detector |
-| <pre>pengcam kw</pre>  | detector |
-| <pre>kengcam swfsc</pre>  | detector |
-| <pre>scallop and flatfish</pre>  | detector |
-| <pre>scallop and flatfish left</pre>  | detector |
-| <pre>scallop netharn</pre>  | detector |
-| <pre>scallop netharn left</pre>  | detector |
+| <pre>motion</pre>  | detect regions of motion in video or time-series images |
+| <pre>pengcam kw</pre>  | Penguin cam full-frame classifier |
+| <pre>pengcam swfsc</pre>  | Penguin cam full-frame classifier |
+| <pre>scallop and flatfish</pre>  | detector for benthic images |
+| <pre>scallop and flatfish left</pre>  | detector for benthic images |
+| <pre>scallop netharn</pre>  | deep learning detector for benthic images |
+| <pre>scallop netharn left</pre>  | deep learning detector for benthic images |
 | <pre>sea lion multi class</pre>  | detects bulls, cows, pups, etc |
 | <pre>sea lion single class</pre>  | detector |
-| <pre>sefsc bw *</pre>  | black-and-white multi-class fish |
+| <pre>sefsc bw 18cls</pre>  | black-and-white 18-class fish detector (oldest, v1) |
+| <pre>sefsc bw 108cls v2</pre>  | black-and-white 108-class fish detector (updated, v2) |
 
 ## Tracking
 
 Run full tracking pipelines on your data.  Appropriate for videos and image sequences that derive from a video.  Tracking involves first running a detection pipeline then performing detection linking to form connected object tracks.
 
+> **Note** some trackers can perform differently on time-series data depending on the annotation framerate selected when you upload or import your dataset. Higer framerates take longer to process, but may produce better results.
+
 | Pipeline | Use case |
 | -------- | -------- |
 | <pre>em tuna</pre>  | tracker |
 | <pre>fish</pre>  | simple fish tracker |
 | <pre>fish.sfd</pre>  | tracker |
-| <pre>generic</pre>  | tracker |
-| <pre>motion</pre>  | tracker |
-| <pre>mouss</pre>  | tracker |
-| <pre>sefsc bw *</pre>  | black-and-white multi-class fish |
+| <pre>generic</pre>  | generic object tracker for bootstrapping groundtruth annotations |
```

Comment:
all pipelines can bootstrap groundtruth annotations this isn't unique to generic detector and generic detector is not something you should encourage people to run over the others necessary, I would just say "Puts classless boxes around arbitrary objects" or the like